app/views/shared/_social_cards.html.erb

```diff
@@ -1,5 +1,5 @@
 <% title ||= "GRID-Arendal Website" %>
-<% description ||= "GRID-Arendal is a Norwegian foundation working closely with the United Nations Environment. We are working on projects all around the world on biodiversity, environmental crime, climate change and Indigenous Peoples." %>
+<% description = description.present? ? strip_tags(description) : "GRID-Arendal is a Norwegian foundation working closely with the United Nations Environment. We are working on projects all around the world on biodiversity, environmental crime, climate change and Indigenous Peoples." %>
```

Comment:
oh OK... maybe I'm with the wrong idea. I'll test it as well and if it works I'll merge! =)